packages/docs-site/src/content/docs/docs/plugins/bitwarden.mdx

```diff
@@ -0,0 +1,144 @@
+
+## Installation & setup
+
+Install the package in the service(s) that will use secrets from Bitwarden.
+
+<TabbedCode packageName="@dmno/bitwarden-plugin" />
+
+-----
+
+After installation, you'll need to initialize the plugin in your dmno config and add a config item to hold your machine account access token. You can explicitly wire the plugin up to the service account token if using multiple tokens at once, or it will be injected by default. It's ok if you have not created the machine account or access token - we'll do that in the next section.
+
+```diff lang="ts" title='.dmno/config.mts'
++import { BitwardenSecretsManagerDmnoPlugin, BitwardenSecretsManagerTypes } from '@dmno/bitwarden-plugin';
+
+// by default, access token will be injected using types
++const bitwardenPlugin = new BitwardenSecretsManagerDmnoPlugin('bitwarden');
+
+// or you can explicitly wire it up by path
++const bitwardenPlugin2 = new BitwardenSecretsManagerDmnoPlugin('bitwarden', {
++  accessToken: configPath('..', 'BWS_TOKEN')
++});
+
+export default defineDmnoService({
+  schema: {
++    BWS_TOKEN: {
++      extends: BitwardenSecretsManagerTypes.machineAccountAccessToken,
++      // NOTE - the type itself is already marked as sensitive 🔐
++    },
+  },
+});
+```
+
+:::tip[Plugin instance IDs]
+You must give each plugin instance a unique id so we can refer to it in other services and the [`dmno` CLI](/docs/reference/cli/plugin/).
+
+In this case we used `bitwarden`, but you can imagine splitting vaults and access, and having multiple plugin instances - for example `bitwarden/prod` for highly sensitive production secrets and `bitwarden/dev` for everything else.
+:::
+
+### Injecting the plugin in monorepo services
+In a monorepo, you are likely managing secrets for multiple services. If you will be using the same service account(s) to access those secrets, you can initialize a plugin instance once in your root service as seen above, and then _inject_ it in child services. Note we must use that same id we set during initialization.
+
+```typescript title='apps/some-service/.dmno/config.mts'
+import { BitwardenSecretsManagerDmnoPlugin } from '@dmno/bitwarden-plugin';
+
+// 💉 inject the already initialized plugin instead of re-initializing it
+const bitwardenPlugin = BitwardenSecretsManagerDmnoPlugin.injectInstance('bitwarden');
+```
+
+
+------------
+
+## Setup Project & Secrets
+
+If you are already using Bitwarden Secrets Manager, you likely aleady have existing [projects](https://bitwarden.com/help/projects/) that contain [secrets](https://bitwarden.com/help/secrets/). If so, now would be a good time to review how they are all organized. If not, you should create at least one project, as each secret can have a parent project it belongs to, and access can be granted to projects rather than managing each secret individually.
```

Comment:
```suggestion
If you are already using Bitwarden Secrets Manager, you likely already have existing [projects](https://bitwarden.com/help/projects/) that contain [secrets](https://bitwarden.com/help/secrets/). If so, now would be a good time to review how they are all organized. If not, you should create at least one project, as each secret can have a parent project it belongs to, and access can be granted to projects rather than managing each secret individually.
```